conftest.py

```diff
@@ -3,14 +3,28 @@
 from page_objects.PageFactory import PageFactory
 from conf import browser_os_name_conf
 from conf import base_url_conf
+from conf import api_example_conf
 from utils import post_test_reports_to_slack
 from utils.email_pytest_report import Email_Pytest_Report
+from endpoints.API_Player import API_Player
 from utils import Tesults
+from utils import interactive_mode
+import argparse
 
 @pytest.fixture
-def test_obj(base_url,browser,browser_version,os_version,os_name,remote_flag,testrail_flag,tesults_flag,test_run_id,remote_project_name,remote_build_name,testname):
+def test_obj(base_url,browser,browser_version,os_version,os_name,remote_flag,testrail_flag,tesults_flag,test_run_id,remote_project_name,remote_build_name,testname,interactivemode_flag):
     "Return an instance of Base Page that knows about the third party integrations"
+
+
     try:
+
+        if interactivemode_flag.lower() == "y":
+
```

Comment:
The whiteline spacing here is off. Why are there empty lines after the if?

Reply:
Fixed